Formula/djl-serving.rb

```diff
@@ -44,9 +44,14 @@ def plist
   test do
     assert_predicate testpath/"#{bin}/djl-serving", :exist?
     assert_predicate testpath/"#{libexec}/lib/serving-0.11.0.jar", :exist?
-    shell_output("sh #{bin}/djl-serving > /dev/null 2>&1 &")
-    sleep 3
+    dir = Dir.mktmpdir('djl-ai')
+    puts dir
+    http_port = free_port
+    puts http_port
+    shell_output("sh -c 'export DJL_CACHE_DIR=$dir; #{bin}/djl-serving > /dev/null 2>&1 &'")
```

Comment:
Shouldn't you be forking a process rather than using `shell_output` for something it's not really designed for here?

Reply:
yeah. I am having trouble with this. Also, I have another doubt. This script starts a process at port 8080. to run the curl command and connect to the script/server, we can't possibly use `free_port`? Am I right ? Or am I missing something here ?